core/src/main/java/nl/tudelft/contextproject/core/screens/GameScreen.java

```diff
@@ -38,20 +50,35 @@
      */
     public GameScreen(final Main main) {
         this.main = main;
-        ColourPalette palette = ColourPalette.standardPalette();
-        player = new Player(palette);
+        player = new Player(ColourPalette.standardPalette(), 100f, 100f);
```

Comment:
Make the constructor accept a List<Player> instead of creating them all for ourselves, to improve testability, if we want convenience access make factory method or overload the constructor